File: src/main/java/net/igneo/icv/entity/wave/WaveRenderer.java
package net.igneo.icv.entity.wave;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class WaveRenderer extends GeoEntityRenderer<WaveEntity> {
    public WaveRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new WaveModel());
    }
}

File: src/main/java/net/igneo/icv/entity/leggings/wave/WaveRenderer.kt
package net.igneo.icv.entity.leggings.wave

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class WaveRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<WaveEntity?>(renderManager, WaveModel())
